Packs/Synapse/Integrations/Synapse/Synapse.py

```diff
@@ -480,30 +464,21 @@ def domain_reputation_command(client, args, good_tag, bad_tag):
             dbot_score=dbot_score
         )
 
-        domain_standard_list.append(domain_standard_context)
-        domain_data_list.append(domain_data)
-
-    readable_output = tableToMarkdown('Domain List', domain_data_list)
-
-    results = CommandResults(
-        readable_output=readable_output,
-        outputs_prefix='Synapse.Domain',
-        outputs_key_field='domain',
-        outputs=domain_data_list,
-        indicators=domain_standard_list
-    )
+        command_results.append(CommandResults(
+            readable_output=tableToMarkdown('Domain List', domain_data),
```

Comment:
Given a single Domain, I feel that Domain List is not a good human-readable title, perhaps Domain Details?